Watchman.Discord/Areas/Initialization/Controllers/InitializationController.cs

```diff
@@ -33,21 +36,42 @@ public InitializationController(IQueryBus queryBus, ICommandBus commandBus, Mess
         public void Init(DiscordRequest request, Contexts contexts)
         {
             ResponsesInit();
+            MuteRoleInit(contexts);
         }
 
         private void ResponsesInit()
+        {
+            var responsesInBase = GetResponsesFromBase();
+            var defaultResponses = GetResponsesFromFile();
+
+            var responsesToAdd = defaultResponses.Where(def => responsesInBase.All(@base => @base.OnEvent != def.OnEvent));
+
+            var command = new AddResponsesCommand(responsesToAdd);
+            _commandBus.ExecuteAsync(command);
+        }
+
+        private IEnumerable<DomainModel.Responses.Response> GetResponsesFromBase()
         {
             var query = new GetResponsesQuery();
-            var responsesInBase = queryBus.Execute(query).Responses;
+            var responsesInBase = _queryBus.Execute(query).Responses;
+            return responsesInBase;
+        }
 
-            if (!responsesInBase.Any())
+        private IEnumerable<DomainModel.Responses.Response> GetResponsesFromFile()
+        {
+            var fileContent = File.ReadAllText(PATH_TO_RESPONSES_FILE);
```

Comment:
lepiej takie coś robić jako embedded resource z pomocą pliku .resx

embedded resource => przy kompilacji plik wchodzi w część DLL, przez co masz pewność że 
1 zawsze będziesz mieć do niego dostęp 
2 nikt go nie podmieni 
3 nikt go nie ukradnie, do tego byłby potrzebny reverse engineering 

+łatwo się do niego dostać

Reply:
Ale to już chyba przy osobnym Tasku, bo trochę tu będzie przepisywania